porting/examples/linux/Makefile

```diff
@@ -0,0 +1,179 @@
+  $(PROJ_ROOT)/nimble/host/util/src/addr.c \
+  $(PROJ_ROOT)/nimble/host/store/ram/src/ble_store_ram.c \
+  $(PROJ_ROOT)/nimble/transport/socket/src/ble_hci_socket.c \
+  $(NULL)
+
+# Source files for common crypto
+SRC_FILES += \
+  $(PROJ_ROOT)/ext/tinycrypt/src/aes_decrypt.c \
+  $(PROJ_ROOT)/ext/tinycrypt/src/aes_encrypt.c \
+  $(PROJ_ROOT)/ext/tinycrypt/src/cmac_mode.c \
+  $(PROJ_ROOT)/ext/tinycrypt/src/ecc.c \
+  $(PROJ_ROOT)/ext/tinycrypt/src/ecc_dh.c \
+  $(PROJ_ROOT)/ext/tinycrypt/src/utils.c \
+  $(NULL)
+
+# Source files for demo app
+SRC_FILES += \
+  ./main.c \
+  ./ble.c \
+  $(NULL)
+
+
+# Include folders common to all targets
+INC_FOLDERS += \
+  ./ \
+  ./include \
+  $(PROJ_ROOT)/porting/npl/linux/include \
+  $(PROJ_ROOT)/porting/npl/linux/src \
+  $(PROJ_ROOT)/porting/nimble/include \
+  $(PROJ_ROOT)/nimble/include \
+  $(PROJ_ROOT)/nimble/host/include \
+  $(PROJ_ROOT)/nimble/host/mesh/include \
+  $(PROJ_ROOT)/nimble/host/services/ans/include \
+  $(PROJ_ROOT)/nimble/host/services/gap/include \
+  $(PROJ_ROOT)/nimble/host/services/gatt/include \
+  $(PROJ_ROOT)/nimble/host/services/ias/include \
+  $(PROJ_ROOT)/nimble/host/services/lls/include \
+  $(PROJ_ROOT)/nimble/host/services/tps/include \
+  $(PROJ_ROOT)/nimble/host/util/include \
+  $(PROJ_ROOT)/nimble/host/store/ram/include \
+  $(PROJ_ROOT)/nimble/controller/include \
+  $(PROJ_ROOT)/nimble/transport/ram/include \
+  $(PROJ_ROOT)/nimble/transport/socket/include \
+  $(PROJ_ROOT)/nimble/drivers/nrf52/include \
+  $(PROJ_ROOT)/ext/tinycrypt/include \
+  $(NULL)
+
+# Libraries common to all targets
+LIB_FILES += 
+
+# C flags common to all targets
+#CFLAGS += -D_POSIX_C_SOURCE=199309L   # minimum for siginfo
+#CFLAGS += -D_POSIX_C_SOURCE=200112L   # minimum for timedlock
+#CFLAGS += -D_POSIX_C_SOURCE=200809L   # minimum for mutexattr
+#CFLAGS += -D_XOPEN_SOURCE=700         # minimum for mutexattr
+CFLAGS += -D_GNU_SOURCE                # Allows use of full pthread API
+CFLAGS += -DCONFIG_GPIO_AS_PINRESET
+CFLAGS +=  -Wall -Werror -Og -g3
+
+# keep every function in separate section, this allows linker to discard unused ones
+CFLAGS += -ffunction-sections -fdata-sections -fno-strict-aliasing
+CFLAGS += -fno-builtin --short-enums
+
+# C++ flags common to all targets
+CXXFLAGS += 
+
+# Assembler flags common to all targets
+ASMFLAGS += 
+
+# Linker flags
+LDFLAGS += -lpthread -lrt -lstdc++
+
+.PHONY: $(TARGETS) default all clean help flash
+
+# Default target - first one defined
+default: $(TARGETS)
+
+# Print all targets that can be built
+help:
+	@echo following targets are available:
+	@echo 	nrf52832_xxaa
+
+TEMPLATE_PATH := $(NRF5_SDK_ROOT)/components/toolchain/gcc
+
+include Makefile.common
+
+$(foreach target, $(TARGETS), $(call define_target, $(target)))
+
+# Flash the program
+flash: $(OUTPUT_DIRECTORY)/nrf52832_xxaa.hex
+	@echo Flashing: $<
+	nrfjprog --program $< -f nrf52 --sectorerase
+	nrfjprog --reset -f nrf52
```

Comment:
these makefiles are obviously copied from nRF5 SDK - we cannot have it like this in repo. I'd expect that `Makefile` for Linux port is similar to what we have for dummy example

Reply:
Ok, I'll adapt the dummy makefile.